db.go

```diff
@@ -36,12 +36,6 @@ const (
 // All data access is performed through transactions which can be obtained through the DB.
 // All the functions on DB will return a ErrDatabaseNotOpen if accessed before Open() is called.
 type DB struct {
-	// Put `stats` at the first field to ensure it's 64-bit aligned. Note that
-	// the first word in an allocated struct can be relied upon to be 64-bit
-	// aligned. Refer to https://pkg.go.dev/sync/atomic#pkg-note-BUG. Also
-	// refer to discussion in https://github.com/etcd-io/bbolt/issues/577.
-	stats Stats
```

Comment:
sorry to say this, but did you read the comment? :)

Reply:
Yes.

Reply:
do you guys have a raspi or any other arm32 device sitting around to validate this? I'm not so worried about the stats pointer, but rather the `sync.Once` or `sync.Pool` could become problematic.

Reply:
Just quickly ran the tests on your branch on my pi - so everything works as intended. Sorry I'm in the middle of a move, luckily I had it plugged in and could ssh into it. Not sure whether you have your QEMU setup still somewhere...

Reply:
> do you guys have a raspi or any other arm32 device sitting around to validate this?

`qemu-user-static` might be good enough to verify this. https://github.com/etcd-io/bbolt/issues/577#issuecomment-1762744870

I think we might need to setup a workflow to guarantee it won't be broken in future.



> but rather the `sync.Once` or `sync.Pool` could become problematic.

The known issue https://pkg.go.dev/sync/atomic#pkg-note-BUG seems only specific to `sync.atomic `package. But I agree that we should verify it anyway.

Reply:
> I think we might need to setup a workflow to guarantee it won't be broken in future.

On it.

Reply:
thx @ivanvc  pls refer to https://github.com/etcd-io/bbolt/issues/665